src/wp-includes/load.php

```diff
@@ -1437,6 +1440,18 @@ function is_ssl() {
  * @return int An integer byte value.
  */
 function wp_convert_hr_to_bytes( $value ) {
+	return wp_hr_bytes( $value );
```

Comment:
```suggestion
	_deprecated_function( __FUNCTION__, '6.1.0', 'get_post()' );
	return wp_hr_bytes( $value );
```

Reply:
thank you! What is the `get_post()` here for? Should this say `'6.1.0', 'wp_hr_bytes()' );` instead?